Hazel/src/Hazel/Application.cpp

```diff
@@ -29,6 +29,7 @@ namespace Hazel {
 		PushOverlay(m_ImGuiLayer);
 	}
 
+
```

Comment:
This is just an extra open line.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/66/commits/d91579dd7ff16b35eb4a684faa738f5bf1638b3d